### PR TITLE
feat: Nervos Network and YokaiDex

### PR DESCRIPTION
**Pull request by developerfred**
## Description

Nervos Network 
YokaiSwap [https://defillama.com/protocol/yokaiswap]

Closes # (NO_ISSUE)

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

File: packages/mask/src/plugins/Trader/pipes.ts [hunk 1]
```diff
@@ -128,6 +129,8 @@ export function resolveTradeProviderLink(tradeProvider: TradeProvider, networkTy
             return 'https://www.trisolaris.io/#/swap'
         case TradeProvider.WANNASWAP:
             return 'https://wannaswap.finance/exchange/swap'
+        case TradeProvider.YOKAISWAP:
+            return 'https://wannaswap.finance/exchange/swap'
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/pipes.ts]: Not correct

File: packages/mask/src/plugins/Trader/pipes.ts [hunk 2]
```diff
@@ -176,6 +179,7 @@ export function resolveTradePairLink(tradeProvider: TradeProvider, address: stri
                 case NetworkType.Fuse:
                 case NetworkType.Metis:
                 case NetworkType.Optimistic:
+                case NetworkType.Nervos:
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/pipes.ts]: These code will be remove, see #5838

File: packages/mask/src/plugins/Trader/settings.ts [hunk 3]
```diff
@@ -91,6 +92,8 @@ export function getCurrentTradeProviderGeneralSettings(tradeProvider: TradeProvi
             return trisolarisSettings
         case TradeProvider.WANNASWAP:
             return wannaswapSettings
+        case TradeProvider.YOKAISWAP:
```
Comment (UncleBill) [inline packages/mask/src/plugins/Trader/settings.ts]: These code will be remove, see #5838

File: packages/mask/src/plugins/Trader/pipes.ts (hunk 2 above)
Comment (developerfred) [reply packages/mask/src/plugins/Trader/pipes.ts]: Ok, I will following this issue #5883

File: packages/mask/src/plugins/Trader/settings.ts (hunk 3 above)
Comment (developerfred) [reply packages/mask/src/plugins/Trader/settings.ts]: ok

**Review by Jack-Works — APPROVED** [review]

**Comment by guanbinrui** [comment]
I'm closing this PR. Cause it doesn't belong to any bounty issue.

**Pull request closed without merging**